<?xml version="1.0"?>
<launch>
  <node pkg="rosbag" type="play" name="play" args="--clock -l $(find medicine_calendar)/bag/2020_medicine.bag"/>
  <node name="case_disturbance" pkg="medicine_calendar" type="case_disturbance" output="screen">
    <param name="threshold_calendar" value="0.01"/> <!--値が小さいほど平面の条件が小さくなる-->
    <param name="calendar_number_neighbors" value="200"/>
    <param name="cluster_tolerance" value="0.01"/>
    <param name="max_cluster_size" value="100000"/>
    <param name="min_cluster_size" value="200"/>
  </node>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find medicine_calendar)/rviz/medicine_calendar.rviz" required="true" output="screen" />
  <rosparam command="load" file="$(find medicine_calendar)/config/locations.yaml" ns="medi_calendar" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="medicine_calendar" args="/medi_calendar/medicine_calendar"/>
  <!--<node pkg="medicine_calendar" name="move_medicine_calendar" type="move_medicine_calendar.py"/>-->
</launch>
